lib/spack/docs/getting_started.rst

```diff
@@ -1580,6 +1590,13 @@ as the project providing Git support on Windows. This is additionally the recomm
 for installing Git on Windows, a link to which can be found above. Spack requires the
 utilities vendored by this project.
 
+"""
```

Comment:
I thought about this a little, technically this is a soft dependency for Spack on Windows as it is possible to get things on their feet without 7zip. Can we make an "Optional but strongly suggested" or similar category to list this and other steps like the Clingo pip installation?

Reply:
> I wasn't able to make any progress without this

The only context in which 7zip is utilized is to extract `.Z` archives, and 7zip can be installed from Spack.
It's also never imported, we run a non fatal `which` command, and if that fails to find 7zip, we inform the users they should `spack install 7zip`. 

It's worth mentioning in this doc, but at the very least we need to point out that users should install 7zip via Spack or add it to the path from where ever they've installed it pre-Spack.